R/messages.R

```diff
@@ -49,3 +49,7 @@ messages <- list(
 
   errorEnumNotAllNames = "The enumValues has some but not all names assigned. They must be all assigned or none assigned"
 )
+
+formatNumerics <- function(numerics, digits = 5, nsmall = 2) {

```

Comment:
@msevestre I think this is what you meatn... I put it in the messages.R as it is used for formatting messages, right?